main/src/main/java/org/mobilitydata/gtfsvalidator/notice/FeedInfoLangAndAgencyLangMismatchNotice.java

```diff
@@ -18,6 +18,11 @@
 
 import com.google.common.collect.ImmutableMap;
 
+/**
+ * {@code agency.agency_lang} and {@code feed_info.feed_lang} do not match
+ *
+ * <p>Severity: {@code SeverityLevel.WARNING}
```

Comment:
Let's make it an ERROR. If feed gives different agency_lang and feed_lang, the intended meaning of that is undefined. Google rejects such feeds.

Reply:
Let's discuss in a new issue/PR if needed so we can get this merged - it's getting hard to manage this PR due to length.

Reply:
New issue on this matter: https://github.com/MobilityData/gtfs-validator/issues/762